ZukoAzula2016/src/org/team1540/zukoazula/DriveCode.java

```diff
@@ -25,19 +26,25 @@
 
     private static final FloatIO driveEncoder = leftCANs[0].modEncoder().getEncoderPosition();
 
-    private static final ArbitratedFloat leftInput = ZukoAzula.behaviors.addFloat();
-    private static final ArbitratedFloat rightInput = ZukoAzula.behaviors.addFloat();
+    private static final BehaviorArbitrator behaviors = new BehaviorArbitrator("Behaviors");
+    private static final Behavior autonomous = behaviors.addBehavior("Autonomous", FRC.inAutonomousMode());
```

Comment:
But the problem is that autonomous and teleop _are_ used elsewhere, they are just used as parts of different BehaviorArbitrators because those subsystems need extra modes. And "Pit Mode" so far is only a thing that matters for drive code.